src/olympia/scanners/tests/test_admin.py

```diff
@@ -84,6 +84,12 @@ def test_formatted_addon_without_version(self):
 
         assert self.admin.formatted_addon(result) == '-'
 
+    def test_guid(self):
+        version = version_factory(addon=addon_factory())
+        result = ScannerResult(version=version)
+
+        assert self.admin.guid(result) == version.addon.guid
```

Comment:
Could you please add a test case for a result without a version?

Reply:
6dc44fe